numba/npyufunc/dufunc.py

```diff
@@ -52,6 +63,10 @@ def __init__(self, py_func, **kws):
             kws.pop('identity', None))
         super(DUFunc, self).__init__(dispatcher, **kws)
         self._install_type()
+        self._kernel = type('DUFuncKernel_' + self.ufunc.__name__,
```

Comment:
The class name is indifferent, I don't think there's any place in the code that cares about it.